store/iavlstore_test.go

```diff
@@ -144,6 +147,81 @@ func testNormal(t *testing.T) {
 	diskDb.Close()
 }
 
+func testGetTreeAfterFlush(t *testing.T) {
+	diskDb := getDiskDb(t, "testGetTreeAfterFlush")
+	store, err := NewIAVLStore(diskDb, 0, 0, flushInterval)
+	require.NoError(t, err)
+	var s string
+	var flushedVersion, latestVersion int64
+	for i := int64(1); i <= flushInterval; i++ {
+		s = strconv.FormatInt(i, 10)
+		store.Set([]byte("key"+s), []byte("value"+s))
+
+		_, latestVersion, err = store.SaveVersion(nil)
+
+		require.NoError(t, err)
+		require.Equal(t, i, latestVersion)
+
+	}
+	flushedVersion = latestVersion
+	store.Set([]byte("k"), []byte("v"))
+	_, latestVersion, err = store.SaveVersion(nil)
+	require.NoError(t, err)
+	//
+	it, err := store.tree.GetImmutable(flushedVersion)
+	require.NoError(t, err)
+	require.Equal(t, flushedVersion, it.Size())
```

Comment:
Why is this `it.Size()` check here?

Reply:
My intention is to check a number of elements in the Immutable tree.
Since we set only one key-value on each SaveVersion, Tree version should equal to numbers of the element.
Is this irrelevant to the test?

Reply:
Relevant enough I suppose, add a comment to the test to explain this.